affiliate/src/productCard/Edit.jsx

```diff
@@ -61,6 +67,20 @@ const Edit = ({ attributes, setAttributes, productSearchPageUrl }) => {
     (displayDescription) => setAttributes({ displayDescription }),
     [setAttributes],
   );
+
+  const [bannerTextValue, setBannerTextValue] = useState(attributes.bannerText);
```

Comment:
I assumed that we didn't want to have a default text, except for whatever has been saved already. So, we wouldn't want something like:
```js
useState(attributes.bannerText || 'Best Overall')
```